collect_app/src/main/java/org/odk/collect/android/widgets/GridMultiWidget.java

```diff
@@ -67,10 +66,8 @@
 @SuppressLint("ViewConstructor")
 public class GridMultiWidget extends QuestionWidget implements MultiChoiceWidget {
 
-    // The RGB value for the orange background
-    public static final int ORANGE_RED_VAL = 255;
-    public static final int ORANGE_GREEN_VAL = 140;
-    public static final int ORANGE_BLUE_VAL = 0;
+    // Using Resource Color highContrastHighlight instead of RGB Values to guarantee consistency
```

Comment:
I think the code is self-explanatory and does not need a comment to explain the usage of the color resource. So, this comment can be dropped as well

Reply:
Removed the redundant comment.

Thanks @shobhitagarwal1612